### PR TITLE
Update for March 29, 2019 hot fix

### PR DESCRIPTION
**Pull request by jfrontain**
## This PR is a:

- [ ] New topic
- [ x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will update the relevant release notes for product versions that are affected by Google's recent deprecation of Google Image Charts.

## Additional information

List all affected URLs 

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4084) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4084) before we can accept your contribution.<br/><hr/>**Jeanne Frontain** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md [hunk 1]
```diff
@@ -29,6 +29,12 @@ Follow these steps to download and apply this patch:
 
 
 
+## Apply patch Admin Dashboard Image-Charts to address deprecation of Google Image Charts
+
+Magento 2.x currently uses Google Image Charts to render static charts in Magento 2.x Admin dashboards. Google stopped supporting Google Image Charts on March 14, 2019, and Magento is providing the Admin Dashboard Image-Charts  patch to replace Google Image Charts with the Image-Charts free service. Users of Magento 2.x deployments will not be able to view static charts in Magento 2.x instances unless they download and apply this patch.
```
Comment (keharper) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]:
```suggestion
Magento 2.x currently uses Google Image Charts to render static charts in Admin dashboards. Google stopped supporting Google Image Charts on March 14, 2019, and Magento is providing the Admin Dashboard Image-Charts  patch to replace Google Image Charts with the Image-Charts free service. To re-enable static charts, you must download and apply this patch. 
```

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md [hunk 2]
```diff
@@ -29,6 +29,12 @@ Follow these steps to download and apply this patch:
 
 
 
+## Apply patch Admin Dashboard Image-Charts to address deprecation of Google Image Charts
+
+Magento 2.x currently uses Google Image Charts to render static charts in Magento 2.x Admin dashboards. Google stopped supporting Google Image Charts on March 14, 2019, and Magento is providing the Admin Dashboard Image-Charts  patch to replace Google Image Charts with the Image-Charts free service. Users of Magento 2.x deployments will not be able to view static charts in Magento 2.x instances unless they download and apply this patch. 
+ 
+This patch is now available for download. See  [Switch from deprecated Google Image Charts to Image-Charts for Magento](https://support.magento.com/hc/en-us/articles/360024850172) for information on downloading and applying this patch.
```
Comment (keharper) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]:
```suggestion
See [Switch from deprecated Google Image Charts to Image-Charts for Magento](https://support.magento.com/hc/en-us/articles/360024850172) for information on downloading and applying this patch. 
```

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md [hunk 3]
```diff
@@ -32,6 +32,13 @@ Follow these steps to download and apply this patch:
 6. Refresh the cache from the Admin (**System** > **Cache Management**).
 
 
+## Apply patch Admin Dashboard Image-Charts  to address deprecation of Google Image Charts
```
Comment (keharper) [inline guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]:
```suggestion
## Apply the Admin Dashboard Image-Charts patch  to address the deprecation of Google Image Charts
```

File: guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md (hunk 1 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.17CE.md]: done

File: guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md (hunk 3 above)
Comment (jfrontain) [reply guides/v2.1/release-notes/ReleaseNotes2.1.17EE.md]: done

**Comment by jfrontain** [comment]
running tests

**Pull request merged**